Blocks/src/Template/Admin/Catalog/chooser.ctp

```diff
@@ -0,0 +1,58 @@
+<div class="row-fluid">
+	<div class="span12">
+		<?php
+		echo __d('croogo', 'Sort by:');
+		echo ' ' . $this->Paginator->sort('id', __d('croogo', 'Id'), array('class' => 'sort'));
+		echo ', ' . $this->Paginator->sort('title', __d('croogo', 'Title'), array('class' => 'sort'));
+		echo ', ' . $this->Paginator->sort('created', __d('croogo', 'Created'), array('class' => 'sort'));
+		?>
+	</div>
+</div>
+
+<?php foreach ($items as $item): ?>
+	<div class="row-fluid">
+		<h2><?= $item->title; ?></h2>
+		<em><strong><?= h(__d('croogo', 'Class name')); ?>: </strong> <?= $item->className; ?></em>
```

Comment:
elements intended to have styling effects like `<em>` and `<strong>` are better avoided in favour of class names, so they can be better controlled via CSS.